package.json

```diff
@@ -16,7 +16,7 @@
     "test:dev:verbose": "jest --watch \"^((?!slow).)*$\"",
     "build": "tsc && cp -R ./src/code-templates/ ./.dist/src/code-templates",
     "build:watch": "tsc --watch",
-    "start:dev:reload": "nodemon ./src/code-generator/index.js"
+    "start:dev:reload": "nodemon ./bin/cli.ts"
```

Comment:
also notice I use `.ts` here and not `.js` as it missing...

Reply:
@rluvaton Why would it work without any just-in-time transpiler like ts-noe?